spec/graphql/interface_type_spec.rb

```diff
@@ -149,4 +149,50 @@
       assert_equal expected_result, result["data"]
     end
   end
+
+  describe "#get_possible_type" do
+    let(:query_string) {%|
+      query fav {
+        favoriteEdible { fatContent }
+      }
+    |}
+
+    let(:query) { GraphQL::Query.new(Dummy::Schema, query_string) }
+    let(:dummy_query_context) { OpenStruct.new(schema: Dummy::Schema, query: query) }
```

Comment:
Not sure if there's a preferred/easier way of getting a "real" context for tests like these.